### DISCUSSION TITLE
Eigenstrain in Phase Field Models

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#20726] [Q&A Modules: General] [answered]
Dear MOOSE Community

I am working with the martensite transformation simulation for which my model is 2D and I need to add variable eigenstrain for which I have specified the eigen_base as "0.1 0 0 0 -0.1 0 0 0 0" as I wanted to incorporate a vector "0.1 -0.1 0" but I am not sure that I have implemented it correctly.So any help in this regard will be beneficial.

Many Thanks 
Pushkar

**Comment by GiudGiud** [comment]
Hello

Any benchmarks you know of he could check his implementation on?
@SudiptaBiswas @jiangwen84 

Guillaume

**Reply by ppandit95** [reply]
Thanks @GiudGiud

**Comment by SudiptaBiswas** [comment, marked as answer]
@ppandit95 You can visualize the deformation of your simulation block to make sure you have implemented the eigen-strain correctly. You can find an example in Section 5 of this paper, https://www.sciencedirect.com/science/article/pii/S0168874X20301165. 

Also, there are multiple examples in,  https://github.com/idaholab/moose/tree/next/modules/combined/examples/phase_field-mechanics.

**Reply by ppandit95** [reply]
Thanks @SudiptaBiswas for directing to relevant sources ...
